swan-lake/learn-the-platform/run-in-the-cloud/function-as-a-service/azure-functions.md

```diff
@@ -1,379 +1,148 @@
 
-The following Ballerina code gives an example of using an HTTP trigger to invoke the function and uses multiple output bindings to return the HTTP response and queue output binding to write an entry to a queue.
+## Set up the prerequisites
 
-First, create a queue to hold the outputs of the function by accessing the storage account that was created alongside the function app in the prerequisites. Select **Queues** in the sidebar in the storage accounts. Click the **Add queue** button, and enter the same value as the value of the `queueName` property in the below `QueueOutput` annotation.
+Follow the instructions in the sections below to set up the prerequisites.
 
-```ballerina
-import ballerina/http;
-import ballerinax/azure_functions as af;
+### Set up Azure Functions 
 
-public type Person record {
-    string name;
-    int age;
-};
+Follow the steps below to set up the Azure Functions.
 
-service / on new af:HttpListener() {
-    resource function post queue(@http:Payload Person person) returns [@af:HttpOutput http:Created, @af:QueueOutput {queueName: "people"} string] {
-        http:Created httpRes = {
-            body: person.name + " Added to the Queue!"
-        };
-        return [httpRes, person.name + " is " + person.age.toString() + " years old."];
-    }
-}
-```
+1. Install the <a href="https://docs.microsoft.com/en-us/cli/azure/install-azure-cli?view=azure-cli-latest" target="_blank">Azure CLI</a>.
+2. Install and configure the [Functions Core Tools](https://learn.microsoft.com/en-us/azure/azure-functions/functions-run-local?tabs=v4%2Clinux%2Ccsharp%2Cportal%2Cbash#install-the-azure-functions-core-tools).
 
-Execute the `bal build --cloud="azure_functions"` command on the package directory to build the package. Then, execute the `func azure functionapp publish <function_app_name> --script-root target/azure_functions ` command shown in the Ballerina build output to deploy it.
+### Create the app
 
-Now, the deployed Azure Function can be tested by invoking it using an HTTP client such as cURL. 
+Follow the steps below to create the Azure function app.
 
-```bash
-$ curl --header "Content-Type: application/json" \
-  --request POST \
-  --data '{"name":"Jack","age":21}' \
-  "https://<function_app_name>.azurewebsites.net/queue"
-Jack Added to the Queue!
-```
+1. Execute the `az login` command on the CLI to log in to the Azure CLI.
+2. Create an <a href="https://docs.microsoft.com/en-us/azure/azure-functions/functions-create-function-app-portal" target="_blank">Azure Function app</a> with the default option of creating a resource group automatically and the requirements below.
+>**Note:** Make sure to remember the function application name and storage account name as they will be required in the code samples.
+      - Runtime stack - `Java 11`
```

Comment:
These bullet points are not properly displayed in GH view file mode. 

<img width="1058" alt="image" src="https://github.com/ballerina-platform/ballerina-dev-website/assets/16300038/8537d496-b4b9-4782-a63c-4dc3687bcc00">

Reply:
Fixed.